leaf-ui/js/onFunctions.js

```diff
@@ -734,21 +734,12 @@ $('#colorblind-mode-isOn').on('click', function(){ //turns off colorblind mode
  * @param {joint.dia.Cell} cell
  */
 function createIntention(cell) {
+    var newIntentionBBM = new IntentionBBM({})
+    cell.set('intention', newIntentionBBM);
+    cell.attr('.funcvalue/text', ' ');
 
-    var name = cell.attr(".name/text") + "_" + Intention.numOfCreatedInstances;
-    cell.attr(".name/text", name);
-
-    // create intention object
-    var type = cell.attributes.type;
-    var intention = new Intention('-', type, name);
-    model.intentions.push(intention);
-
-    // create intention evaluation object
-    var intentionEval = new UserEvaluation(intention.nodeID, '0', '(no value)');
-    analysisRequest.userAssignmentsList.push(intentionEval);
-
-    cell.attributes.nodeID = intention.nodeID;
-
+    // create intention evaluation object and add it to userEvaluationList    
+    graph.get('userEvaluationList').push(new UserEvaluationBBM({intentionID: newIntentionBBM.cid, absTP: '0'}));
```

Comment:
Sorry, can you clarify why this is happening? I thought we want to start with an empty list?

Reply:
We removed this, thank you!